tools/build.py

```diff
@@ -194,6 +196,7 @@ def build_options_append(cmakeopt, cliarg):
     build_options_append('FEATURE_MEM_STRESS_TEST', arguments.mem_stress_test)
     build_options_append('FEATURE_PROFILE', arguments.profile)
     build_options_append('FEATURE_REGEXP_STRICT_MODE', arguments.regexp_strict_mode)
+    build_options_append('REGEXP_RECURSION_LIMIT', arguments.regexp_recursion_limit)
```

Comment:
To follow the pattern, shouldn't it be `FEATURE_REGEXP_RECURSION_LIMIT`?

Reply:
Not necessarily. `FEATURE_` options are on/off toggles. Recursion limit is a numeric value, so one can argue for not prefixing it with `FEATURE_`. Another such option is `MEM_HEAP_SIZE_KB`.